providers/capa/standard/capa_suite_test.go

```diff
@@ -0,0 +1,63 @@
+package standard
+
+import (
+	"context"
+	"path"
+	"testing"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	cr "sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/giantswarm/clustertest"
+	"github.com/giantswarm/clustertest/pkg/application"
+	"github.com/giantswarm/clustertest/pkg/logger"
+	"github.com/giantswarm/clustertest/pkg/utils"
+	"github.com/giantswarm/clustertest/pkg/wait"
+)
+
+const KubeContext = "capa"
+
+var (
+	framework *clustertest.Framework
+	cluster   *application.Cluster
+)
+
+func TestCAPAStandard(t *testing.T) {
+	var err error
+	ctx := context.Background()
+
+	logger.LogWriter = GinkgoWriter
+
+	framework, err = clustertest.New(KubeContext)
+	if err != nil {
+		panic(err)
+	}
+
+	cluster = application.NewClusterApp(utils.GenerateRandomName("t"), application.ProviderAWS).
+		WithAppValuesFile(path.Clean("./test_data/cluster_values.yaml"), path.Clean("./test_data/default-apps_values.yaml"))
+
+	BeforeSuite(func() {
+		logger.Log("Workload cluster name: %s", cluster.Name)
+
+		applyCtx, cancelApplyCtx := context.WithTimeout(ctx, 20*time.Minute)
+		defer cancelApplyCtx()
+
+		client, err := framework.ApplyCluster(applyCtx, cluster)
+		Expect(err).ToNot(HaveOccurred())
```

Comment:
Just syntax sugar so feel free to ignore but I think `NotTo` is used a lot more than `ToNot`.

Reply:
I actually got that suggestion from running `ginkgolinter` against the codebase. It seems it treats them both the same so I'm happy to switch them all :)